zmon_worker_monitor/zmon_worker/notifications/hipchat.py

```diff
@@ -19,8 +21,15 @@ def notify(cls, alert, *args, **kwargs):
 
         color = 'green' if alert and not alert.get('is_alert') else kwargs.get('color', 'red')
 
+        zmon_host = kwargs.get('zmon_host', cls._config.get('zmon.host'))
```

Comment:
I'd move this (excluding `message_text = ...` of course) inside the `if kwargs.get('link', False):`